SH4RE/SH4RE/CreateListing/CreateListingView.swift

```diff
@@ -271,26 +272,31 @@ struct CreateListingView: View {
                 calAvail = allDays.filter{ !calendar.isDateInWeekend($0) }
             }
         }
-        let listingFields = ["Title": title, "Description" : description, "Price" : cost, "Category" : categorySelection, "Availability": calAvail, "Address": ["postalCode": postalCode, "latitude": lat, "longitude": lon], "UID": getCurrentUserUid()] as [String : Any]
-        let documentID = documentWrite(collectionPath: "Listings", data: listingFields)
-        
-        // upload images and add paths to data fields
-        var index = 1
-        var imgPath = ""
-        var arrayImgs:[String] = []
-        for pic in pictures {
-            imgPath = "listingimages/" + documentID + "/" + String(index) + ".jpg"
-            arrayImgs.append(imgPath)
-            storageManager.upload(image: pic, path: imgPath)
-            index += 1
-        }
-        if (documentUpdate(collectionPath: "Listings", documentID: documentID, data: ["image_path" : arrayImgs])) {
-            NSLog("error");
-        }
-        showPostAlertX = true
-
-        //reset inputs
-        resetInputs()
+        getCurrentUser(completion: { user in
```

Comment:
what is this change?

Reply:
this is a change from @bbryanb333 but im assuming its bc he needed the user.name

Reply:
yup